### PR TITLE
bugfix: sqlserver lock_table missing status placeholder

### PR DESCRIPTION
**Pull request by deung**
fix Error in insert operation sql of [lock_table] datasheet adapted to sqlserver database (#6468)
1. The [lock_table] data table insert statement INSERT_LOCK_SQL_SQLSERVER adds the placeholder [?] corresponding to the [status] field.
2. file  sqlserver.sql [lock_table] create sql add [status] field.
3. Refer to the mysql.sql improve sqlserver.sql.

### CODE REVIEW

**Review by funky-eyes — APPROVED** [review]
LGTM 
Please add your author information and PR information to the 2.x.md files in both the Chinese and English versions of changes under https://github.com/apache/incubator-seata/tree/2.x/changes. You can refer to the content of the registered list for filling in the details.

**Review by xingfudeshi — APPROVED** [review]
LGTM

File: changes/en-us/2.x.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6380](https://github.com/apache/incubator-seata/pull/6380)] fix sql exception when checking for the existence of the UNDO_LOG table on SQL server
 - [[#6385](https://github.com/apache/incubator-seata/pull/6385)] fix the bug where Role.participant does not execute hooks but clears them.
 - [[#6465](https://github.com/apache/incubator-seata/pull/6465)] fix(6257): fix saga mode replay context lost start in 2.x
+- [[#6468](https://github.com/apache/incubator-seata/pull/6469)] fix Error in insert sql of [lock_table] data table to sqlserver database
```
Comment (funky-eyes) [inline changes/en-us/2.x.md]:
```suggestion
- [[#6469](https://github.com/apache/incubator-seata/pull/6469)] fix Error in insert sql of [lock_table] data table to sqlserver database
```

File: changes/zh-cn/2.x.md [hunk 2]
```diff
@@ -35,6 +35,7 @@
 - [[#6380](https://github.com/apache/incubator-seata/pull/6380)] 修复针对sql server检查UNDO_LOG表是否存在时的SQL异常
 - [[#6385](https://github.com/apache/incubator-seata/pull/6385)] 修复Role.Participant不执行hook但会清理的问题
 - [[#6465](https://github.com/apache/incubator-seata/pull/6465)] 修复2.0下saga模式的context replay丢失start问题
+- [[#6468](https://github.com/apache/incubator-seata/pull/6469)] 修复在sqlserver数据库下[lock_table]数据表的插入操作sql中存在的错误
```
Comment (funky-eyes) [inline changes/zh-cn/2.x.md]:
```suggestion
- [[#6469](https://github.com/apache/incubator-seata/pull/6469)] 修复在sqlserver数据库下[lock_table]数据表的插入操作sql中存在的错误
```

**Pull request merged**